.gitattributes

```diff
@@ -7,10 +7,11 @@
 *.fixed linguist-language=Rust
 *.mir linguist-language=Rust
 src/etc/installer/gfx/* binary
-*.woff binary
 src/vendor/** -text
 Cargo.lock linguist-generated=false
 
 # Older git versions try to fix line endings on images, this prevents it.
```

Comment:
It'd be good to tweak the message to include both, as per https://github.com/rust-lang/rust/pull/83162#issuecomment-800011892

Reply:
You think “images” doesn't describe WOFF?

Reply:
Well, would you call a .woff file an "image" in a regular conversation? I think interpreting it an image is as technically correct and helpful as interpreting a text file as a "binary" file, which doesn't really help here. I'm just trying to avoid the shoehorning here.